README.markdown

```diff
@@ -455,7 +455,7 @@ steps to create a new migration script for h are:
    with its current revision, so Alembic knows which migration scripts to run:
 
    ```bash
-   $ alembic -c conf/alembic.ini stamp <revision_id>
+   $ .tox/py36-dev/bin/alembic -c conf/alembic.ini stamp <revision_id>
    ```
```

Comment:
As an aside, I think we should apply https://github.com/hypothesis/h/pull/5388 to the lms app as well to save people the trouble of ending up with a DB that isn't stamped or is stamped with the wrong version.